internal/poadraftapplication/poa_draft_application.go

```diff
@@ -0,0 +1,100 @@
+package poadraftapplication
+
+import (
+	"crypto/sha256"
+	"encoding/json"
+	"fmt"
+
+	"github.com/ministryofjustice/opg-search-service/internal/response"
+)
+
+const AliasName = "poadraftapplication"
+
+type DraftApplicationDonor struct {
+	Name string `json:"name"`
+	Dob string `json:"dob"`
+	Postcode string `json:"postcode"`
+}
+
+type DraftApplication struct {
+	UID *string `json:"uId"`
+	Donor DraftApplicationDonor `json:"donor"`
+}
+
+func (d DraftApplication) Id() string {
+	if d.UID == nil {
+		return "0"
+	}
+	return *d.UID
+}
+
+func (d DraftApplication) Validate() []response.Error {
+	var errs []response.Error
+
+	if d.UID == nil || len(*d.UID) == 0 {
+		errs = append(errs, response.Error{
+			Name: "uId",
+			Description: "field is empty",
+		})
+	}
+
+	return errs
+}
+
+func IndexConfig() (name string, config []byte, err error) {
+	textField := map[string]interface{}{"type": "text"}
+	searchableTextField := map[string]interface{}{"type": "text", "copy_to": "draftApplicationSearchable"}
```

Comment:
Could this just be "searchable" rather than "draftApplicationSearchable"? I feel like the prefix might be redundant since it's always in the context of this index.